prac_05/color_name.py

```diff
@@ -0,0 +1,12 @@
+color_NAMES = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb", "AntiqueWhite2": "#eedfcc",
```

Comment:
Good name for global,consider changing just the format possibly to  COLOR_NAMES :)